commands/translate.ts

```diff
@@ -83,9 +114,17 @@ export const translate = async (
         `I can translate your input to the following languages:\n${languagelist}`
       );
     });
+  } else if (language.search("-") != -1 && language.length <= 11) {
```

Comment:
Can you explain what this number, `11` is here for?

Reply:
The longest modelid is eleven characters (fr-CA-xhtml-TW)

Reply:
Fair enough. I know that will trigger correct behaviour in most cases, like simplified-chinese.

Reply:
And the language that have a dash are traditional-Chinese, simplified-Chinese and French-(Canada) which are all under 11 characters

Reply:
*over 😉

Reply:
ya over. my bad i was half asleep when writting that